packages/amplify_authenticator/lib/src/mixins/authenticator_phone_field.dart

```diff
@@ -0,0 +1,75 @@
+import 'package:amplify_authenticator/src/theme/amplify_theme.dart';
+import 'package:amplify_authenticator/src/utils/country_code.dart';
+import 'package:amplify_authenticator/src/widgets/form_field.dart';
+
+import 'package:flutter/material.dart';
+
+mixin AuthenticatorPhoneField<FieldType,
+        T extends AuthenticatorFormField<FieldType, String, T>>
+    on AuthenticatorFormFieldState<FieldType, String, T> {
+  @override
+  void initState() {
+    selectionValue = '1';
```

Comment:
*nit:* Took me a minute to realize the significance of "1". If `countryCodeStrings` is taken out of the build method, maybe this can be `countryCodeStrings.first`

Reply:
Agreed. Can you repurpose `initialValue` for this?

Also, `super.initState` should always be the first call made when overriding `initState`

Reply:
Yeah - the whole country code data structure needs re-work.